src/Image.php

```diff
@@ -355,7 +355,7 @@ private function getPixelValue($size)
         $size = trim($size);
 
         if (isset($map[substr($size, -2)])) {
-            $size = $size * $map[substr($size, -2)];
+            $size = substr($size, 0, -2) * $map[substr($size, -2)];
```

Comment:
I don't know. Does `floatval()` reliably strip all possible units? Since you rely on `substr($size, -2)` to extract the unit, I figured we might as well rely on `substr()` to extract the unit.